src/guides/v2.3/frontend-dev-guide/layouts/xml-manage.md

```diff
@@ -246,6 +246,18 @@ To set attributes for the HTML `body` tag use the `<attribute>` instruction.
 </page>
 ```
 
+Also from admin site __Layout Update XML__ field of __Design__ section on _Content Edit_ page  set attributes for the HTML `body` tag use the `<attribute>` instruction.
```

Comment:
Information about specifying custom layouts from the Magento Admin is already covered in this section on the ame page: [Create cms-page/product/category-specific layouts](https://devdocs.magento.com/guides/v2.3/frontend-dev-guide/layouts/xml-manage.html#create-cms-pageproductcategory-specific-layouts). Also, this capability applies to customizing layout instructions for other settings in addition to the body attribute, so I think you can safely remove this.

```suggestion
```